Resources/Locale/en-US/deltav/ghost/roles/ghost-role-component.ftl

```diff
@@ -2,4 +2,7 @@ ghost-role-information-nukie-mouse-name = Nuclear Operative Mouse
 ghost-role-information-nukie-mouse-description = A Nuclear Operative reinforcement for the Syndicate.
 ghost-role-information-nukie-mouse-rules = Normal syndicate antagonist rules apply. Work with whoever called you in, and don't harm them.
                                     The crew is allowed to kill you without warning.
-                                    You are allowed to attack the crew and destroy the station without provocation. 
```

Comment:
Can't figure out how to fix this >:(

Reply:
I gotcha, the secret is you have to use a program other than notepad.